Kitura/Proposals/0001-codable-routing.md

```diff
@@ -76,10 +76,6 @@ Utilizing the new API layer, the above code becomes:
 ```swift
 struct Name: Codable {
    let name: String
-   
-   init(name: String) {
```

Comment:
This is only superfluous if you're only going to allow instances to be created using `JSONDecoder().decode()` - which I guess is true in our example ;-)

Reply:
You get a default contructor for structs that matches the stored properties. Check it out: http://swift.sandbox.bluemix.net/#/repl/59e5f9921b6536583365e490

Reply:
TIL ;-)